nexmark-flink/src/main/java/com/github/nexmark/flink/workload/WorkloadSuite.java

```diff
@@ -42,124 +44,131 @@ public class WorkloadSuite {
 	private static final String WARMUP_DURATION_SUFFIX = ".warmup.duration";
 	private static final String KAFKA_BOOTSTRAP_SERVERS = "kafka.bootstrap.servers";
 
-	private final Map<String, Workload> query2Workload;
+	private final Map<String, Map<String, Workload>> query2Workloads;
```

Comment:
@wuchong, IMO, there could be certain benchmark category, because the `WorkloadSuite` is buit from the `nexmark.yaml` configuration file. Hence this could generate the mapping between the category and the mapping between query and workload.